theme/theme_test.go

```diff
@@ -3,8 +3,9 @@ package theme
 import (
 	"testing"
 
-	"fyne.io/fyne"
```

Comment:
This import change is incorrect - the order was right before

Reply:
hmm, i set GoLand to use `goimports` and this is how it seems to think it should be formatted.  Is there some sort of .goimportsrc file support?  Here are my settings:
![Screen Shot 2019-05-05 at 12 48 11 PM](https://user-images.githubusercontent.com/2140511/57197356-55348a80-6f34-11e9-85e9-7321ef9d7273.png)

Reply:
That is the correct setup but I don't think GoLand always fixes imports - sometimes it just highlights a suggestion... I find it's useful to make sure the file checks are "green" as wrong import order turns it "yellow"... (icon at the top right).

Reply:
Ok cool, I thought it was auto-formatting, i'll definitely keep an eye on it.  Thanks!